Assets/GothicVR/Scripts/Data/SkyState.cs

```diff
@@ -56,11 +58,11 @@ public void PresetDay0()
             domeColor0 = fogColor;
             domeColor1 = new Vector3(255.0f, 255.0f, 255.0f);
 
-            layer[0].texName = "SKYDAY_LAYER1_A0.TGA";
-            layer[1].texName = "SKYDAY_LAYER0_A0.TGA";
+            layer[0].texName = "SKYDAY_LAYER0_A0.TGA";
```

Comment:
Are these values also valid on G2? Please add a Try/Catch when you load it and return a Debug.LogError() if not.